### ISSUE TITLE
Logging two curves to a table metrics in AzureML does not visualize well

### ISSUE THREAD

**Issue by SamVanhoutte** [Azure/MachineLearningNotebooks#1011]
Hello all, 

Please consider the following code

```python
# Just a fixed table, normally this comes from a keras training history
run = Run.get_context()
logtable = {
        'epochs': [1, 2, 3, 4, 5],
        'test_accuracy': [0.2, 0.3, 0.45, 0.6, 0.61],
        'train_accuracy': [0.15, 0.28, 0.41, 0.5, 0.59]
        }
run.log_table('Training curve', logtable)
```

When executing this, the table itself is correctly saved to the AzureML run, as you can see in the following screenshot
![image](https://user-images.githubusercontent.com/3539605/84657686-f69e5600-af14-11ea-9030-e88b807a1fc9.png)

However, I would love to see this visualized as two combined line charts in a plot.  But when selecting the chart view, the following error is displayed:
> Chart visualization not available for non-numeric values.

I can work around this, by creating a plot in my training script and save it to the images, but I would love to see them in the metrics, as I believe they have a better fit there.

An example of this is screenshotted below:
![image](https://user-images.githubusercontent.com/3539605/84657889-441ac300-af15-11ea-8914-4df1ad120848.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9190938a-63d6-5b03-ff00-9bc3bf31e27f
* Version Independent ID: 6e9f9c7d-69c3-0a20-f94f-710771255ec6
* Content: [Log ML experiments & metrics - Azure Machine Learning](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-track-experiments)
* Content Source: [articles/machine-learning/how-to-track-experiments.md](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/machine-learning/how-to-track-experiments.md)
* Service: **machine-learning**
* Sub-service: **core**
* GitHub Login: @sdgilley
* Microsoft Alias: **sgilley**

**Comment by YutongTie-MSFT** [comment]
@SamVanhoutte 

Thanks for your feedback! ^^ Please help with the link of document on which above document feedback request is created.

**Comment by SamVanhoutte** [comment]
Here is the link : https://docs.microsoft.com/en-us/azure/machine-learning/how-to-track-experiments

It is not really a documentation issue, but more an SDK/Implementation issue.  I just cannot find the github SDK thing and always arrive here... So if you could point me out the repo for the Python SDK for AzureML, I'll post it there

**Comment by YutongTie-MSFT** [comment]
@SamVanhoutte Hi Sam, I have assigned Daniel from SDK team to handle this problem. Thanks for your information.

**Comment by swanderz** [comment]
@danielsc any update on this?

**Comment by swinner95** [comment]
Hi, @SamVanhoutte today we do not support plotting logged row and table metric columns on the x axis of the charts. We are planning to support this functionality this semester.

**Comment by manuelrames** [comment]
Hi! Was there any advance on this? I am facing the same problem as @SamVanhoutte. I would like to log training and validation loss curves on the same chart to have a visual on the training process on AzureML. Thanks in advance!

**Comment by ManuelSchenk** [comment]
Ok here my solution, this prints "val_loss" together with "train_loss" in one graph with one value per epoch:

` 
        metrics = run.get_metrics()
        plt.plot(list(range(len(metrics.get('val_loss')))), metrics.get('val_loss'), label='val_loss')
        plt.plot(list(range(len( metrics.get('train_loss')))),  metrics.get('train_loss'), label='train_loss')
        plt.legend()
        plt.xlabel("Batches")
        plt.ylabel("Loss")
        run.log_image('overfitting_monitor_img', plot=plt)`
